### PR TITLE
Doc for Calico spanning multiple OpenStack regions

### PR DESCRIPTION
**Pull request by neiljerram**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 073d243f2f4cf6b01b3c260ac4cc274d1206ec28

https://deploy-preview-2378--calico.netlify.com

File: master/usage/openstack/multiple-regions.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+1.  In `/etc/neutron/neutron.conf` on each controller and compute node, add
+
+    ```
+    [calico]
+    openstack_region = <region>
+    ```
+
+    where `<region>` is the name of the region that that node belongs to.
+
```
Comment (fasaxc) [inline master/usage/openstack/multiple-regions.md]: Worth a warning box saying that if the two don't match you'll see symptom XYZ?

File: master/usage/openstack/multiple-regions.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+the label for the 85cc3048... security group.  We know that the 85cc3048... VMs
+_are_ in a different namespace from the a7734e61... VMs, so the
+`namespaceSelector: "all()"` here - which allows the connecting VMs to come
+from any region or namespace - is critical for this policy to work as intended.
+
+You can use any of the
+[labels]({{site.baseurl}}/{{page.version}}/usage/openstack/labels) that
+{{site.prodname}} adds to OpenStack VM endpoints, to identify a set of allowed
+(or denied) clients.  You can also use `nets` or `notNets` instead of
+`selector`, to identify clients by IP address.
```
Comment (fasaxc) [inline master/usage/openstack/multiple-regions.md]: Example looks good but could we give them some more examples of useful selectors and maybe a GlobalNetworkPolicy example?  I'm worried that people will have difficulty in coming up with their own.  For example:

```
Here are some more examples of selectors to get you started:

- To select all VMs in a RegionOne: `projectcalico.org/namespace == 'openstack-RegionOne'`
- To select all VMs that are in a security group named "production" in *either* region: `has(sg-name.projectcalico.org/openstack-production)`

```

Comment (neiljerram) [reply master/usage/openstack/multiple-regions.md]: Thanks, I've switched to Global and worked that text in.

**Comment by fasaxc** [comment]
As discussed, may need to add the new field to FelixConfiguration (and its associated doc).

**Pull request merged**